docs/ibexa_engage/create_campaign.md

```diff
@@ -0,0 +1,31 @@
+---
+description: Create campaign with Ibexa Engage
+edition: experience
+---
+
+# Create campaign
+
+[Campaign]([[= user_doc =]]/ibexa_engage/ibexa_engage/#campaign) is a set of concepts, divided into steps, that the user can configure.
+It includes a welcome screen, an interaction element, a form step, and an exit screen.
+Design, themes, backgrounds are all configured by Qualifio.
```

Comment:
@julitafalcondusza in the user guide there was a remark from @SylvainGuittard , that design, themes can be prepared by the user.